packages/framer-motion/src/animation/animate.ts

```diff
@@ -61,12 +52,10 @@ export type AnimationOptions<V> = (Tween | Spring) &
 export function animate<V>(
     from: MotionValue<V> | V,
     to: V | V[],
-    transition: AnimationOptions<V> = {}
+    transition: Transition & AnimationPlaybackLifecycles<V> = {}
 ): AnimationPlaybackControls {
     const value = isMotionValue(from) ? from : motionValue(from)
     value.start(createMotionValueAnimation("", value, to as any, transition))
 
-    return {
-        stop: () => value.stop(),
-    }
+    return value.animation!
```

Comment:
Is the `!` necessary?

Reply:
Is there a way we can get to the animation in a "safer" way (my first thought was returning it from `start`, but that already returns a promise), so we have more guarantee than "yeah we really started the animation, I'm sure"

Reply:
The idea is in a later iteration for `animate()` to be able to return `GroupPlaybackControls` which can control multiple animations so I could implement that. It shouldn't care if there are none or many underlying animations.

Reply:
Implemented in https://github.com/framer/motion/pull/1993/commits/c926c957c23b28e37c61369f95ce6adffcc1c0f4